pages/en/android/camera-orientation.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Camera orientations
+metadesc: An in-depth guide on how to handle orientations in Android camera apps.
+date: 2020-09-23
+weight: 1
+tags:
+  - camera orientation
+  - large screens
+resources:
+  - title: CameraX Preview Documentation
+    url: https://developer.android.com/reference/androidx/camera/core/Preview
+---
+
+If your Android app uses cameras, there are some special considerations when handling orientations. This article presumes you understand the basic concepts of Android camera2 API. You can read our [blog post](https://medium.com/google-developers/detecting-camera-features-with-camera2-61675bb7d1bf) or [summary](https://developer.android.com/reference/android/hardware/camera2/package-summary) for an overview of camera2. We also recommend that you try writing a simple camera app first before approaching the article.
+
+## Background
+
+Handling orientations in Android camera apps is tricky and needs to take the following factors into consideration:
+
+1. Natural orientation: The display orientation when the device is in the “normal” position for the device design - usually portrait orientation for mobile phones and landscape orientation for laptops.
+2. Sensor orientation: The orientation of the sensor physically mounted on the device.
+3. Display rotation: How much the device is physically rotated from the natural orientation.
+4. Viewfinder size: The size of the viewfinder used for displaying the camera preview.
+5. Image size output by the camera.
```

Comment:
These don't read like an ordered list to me, but rather like a group of things that could be in any order. If that's true, can we make it an unordered list? If not, can it be rewritten so the items are clearly needed in a specific order?

Thanks!
```suggestion
* Natural orientation: The display orientation when the device is in the “normal” position for the device design - usually portrait orientation for mobile phones and landscape orientation for laptops.
* Sensor orientation: The orientation of the sensor physically mounted on the device.
* Display rotation: How much the device is physically rotated from the natural orientation.
&* Viewfinder size: The size of the viewfinder used for displaying the camera preview.
* Image size output by the camera.
```

Reply:
There is an extra ampersand that I cannot take out here. 

> &* Viewfinder size

Reply:
Removed it, thanks for catching!

Reply:
Awesome. Thanks for the suggestion here :)